.github/mergify.yml

```diff
@@ -68,3 +68,28 @@ pull_request_rules:
       request_reviews:
         teams:
           - "@PyTorchLightning/core-lightning"
+
+  - name: Grid.ai approval
+    conditions:
+      - or:
+        - approved-reviews-by=awaelchli
+        - approved-reviews-by=Borda
+        - approved-reviews-by=carmocca
+        - approved-reviews-by=edenlightning
+        - approved-reviews-by=ethanwharris
+        - approved-reviews-by=kaushikb11
+        - approved-reviews-by=rohitgr7
+        - approved-reviews-by=tchaton
+        - approved-reviews-by=williamFalcon
```

Comment:
```suggestion
        - approved-reviews-by=williamFalcon
        - approved-reviews-by=krshrimali
```

Reply:
and @otaj

Reply:
let's add Ota later when he gets familiar with code-base